### DISCUSSION TITLE
[SOLVED] Notifications

### DISCUSSION THREAD

**Discussion by znwmnw13** [martinrotter/rssguard#1222] [General]
Hi there, I recently downloaded RSSGuard today and I was wondering on how I enable notifications on the application for my feeds or certain feeds. If it is not a feature in the RSS feed application, I completely understand that.

Sincerely, znwmnw13

**11/21/23 1:22:44 PM EST**

**Comment by martinrotter** [comment]
Hi. This is controlled by "Ignore notifications...." in feed settings dialog, but at this point there seems to be a related bug where this setting is not properly honored by application. Will fix in future release.

![image](https://github.com/martinrotter/rssguard/assets/1255302/d77f8e5f-867d-432e-933c-5a2f11bb87e0)
